gateway/src/api/latest.rs

```diff
@@ -153,6 +154,15 @@ async fn get_status(Extension(sender): Extension<Sender<BoxedTask>>) -> Response
         .unwrap()
 }
 
+async fn revive_projects(
```

Comment:
From here you should grab `sender` and feed that down `exec::revive`. The only reason why it wasn't there before if because it was initially written to run stand-alone as a subcommand. Now it's part of the API, it should use the same worker queue.